### PR TITLE
Switch to upstream ECS version 3.1.0.1

### PR DESCRIPTION
**Pull request by padthaitofuhot**
This bumps ECS version to 3.1.0.1.

### CODE REVIEW

File: ui/etc/release.conf [hunk 1]
```diff
@@ -8,10 +8,10 @@
 # it is provided by or on behalf of EMC.
 
 release_name="ECS Community Edition"
-release_version="3.1.0.0"
+release_version="3.1.0.1"
 release_product="ECS Software"
 release_artifact="emccorp/ecs-software-3.1.0"
-release_tag="latest"
+release_tag="hf1"
```
Comment (jasoncwik) [inline ui/etc/release.conf]: Is this just for testing until we promote hf1 to latest?

Comment (padthaitofuhot) [reply ui/etc/release.conf]: Email said 3.1.0.1 replaces 3.1.0.0 immediately.  I took that to mean immediately.  So, my plan is to promote 3.1.0.1 to latest ASAP.

**Comment by padthaitofuhot** [comment]
Resolves #374

File: ui/etc/release.conf (hunk 1 above)
Comment (padthaitofuhot) [reply ui/etc/release.conf]: this tag is for hf1 specifically.  I'm using version specific tags now.  HF1 will also be tagged "latest" once it's live.

Comment (padthaitofuhot) [reply ui/etc/release.conf]: In fact, it's already tagged latest, so 2.5.0 instances are getting 3.1.0.1 now as well.

Comment (jasoncwik) [reply ui/etc/release.conf]: coolio

**Pull request merged**